CHANGELOG.md

```diff
@@ -1,6 +1,115 @@
+* **New Resource:** `vcd_vm_placement_policy` that allows creating VM Placement Policies ([#904](https://github.com/vmware/terraform-provider-vcd/pull/904), [#911](https://github.com/vmware/terraform-provider-vcd/pull/911))
+* **New Data Source:** `vcd_vm_placement_policy` that allows fetching existing VM Placement Policies ([#904](https://github.com/vmware/terraform-provider-vcd/pull/904), [#911](https://github.com/vmware/terraform-provider-vcd/pull/911))
+* **New Data Source:** `vcd_provider_vdc` that allows fetching existing Provider VDCs ([#904](https://github.com/vmware/terraform-provider-vcd/pull/904))
+* **New Data Source:** `vcd_vm_group` that allows fetching existing VM Groups, to be able to create VM Placement Policies ([#904](https://github.com/vmware/terraform-provider-vcd/pull/904))
+* **New Resource**: `vcd_org_ldap` that allows configuring LDAP settings for an organization ([#909](https://github.com/vmware/terraform-provider-vcd/pull/909))
+* **New Data Source**: `vcd_org_ldap` that allows exploring LDAP settings for an organization ([#909](https://github.com/vmware/terraform-provider-vcd/pull/909))
+* **New Resource:** `vcd_catalog_access_control` that allows sharing a catalog with users, groups, or Orgs ([#915](https://github.com/vmware/terraform-provider-vcd/pull/915))
+* **New Resource**: `vcd_subscribed_catalog` that allows subscribing to a published catalog ([#916](https://github.com/vmware/terraform-provider-vcd/pull/916))
+* **New Data Source**: `vcd_subscribed_catalog` that allows reading a subscribed catalog ([#916](https://github.com/vmware/terraform-provider-vcd/pull/916))
+* **New Data Source**: `vcd_task` that allows reading a VCD task ([#916](https://github.com/vmware/terraform-provider-vcd/pull/916))
+* Add attribute `metadata_entry` to the following data sources:
+  `vcd_catalog`, `vcd_catalog_media`, `vcd_independent_disk`, `vcd_network_direct`, `vcd_network_isolated`,
+  `vcd_network_isolated_v2`, `vcd_network_routed`, `vcd_network_routed_v2`, `vcd_org`, `vcd_org_vdc`, `vcd_provider_vdc`,
+  `vcd_storage_profile`, `vcd_vapp`, `vcd_vapp_vm`. This new attribute replaces `metadata`
+  to add support of metadata visibility (user access levels), all the available types and domains for every metadata
+  entry. ([#917](https://github.com/vmware/terraform-provider-vcd/pull/917))
+* Add attribute `metadata_entry` to the following resources:
+  `vcd_catalog`, `vcd_catalog_media`, `vcd_independent_disk`, `vcd_network_direct`, `vcd_network_isolated`,
+  `vcd_network_isolated_v2`, `vcd_network_routed`, `vcd_network_routed_v2`, `vcd_org`, `vcd_org_vdc`, `vcd_vapp`,
+  `vcd_vapp_vm`. This new attribute replaces `metadata` to add support of metadata visibility (user access levels),
+  all the available types and domains for every metadata entry. ([#917](https://github.com/vmware/terraform-provider-vcd/pull/917))
+* Add `placement_policy_id` attribute to `vcd_vapp_vm` and `vcd_vm` resource and data source,
```

Comment:
Sometimes the barrier between Feature and Improvement is fuzzy to me, I noticed I placed this inside Features, but  there's
```
Add `vm_placement_policy_ids` attribute to `vcd_org_vdc`
```
in Improvements section.
Should we move this one into Improvements?